buildenv/jenkins/common/build.groovy

```diff
@@ -565,36 +565,51 @@ def add_node_to_description() {
     currentBuild.description = TMP_DESC + "<a href=${JENKINS_URL}computer/${NODE_NAME}>${NODE_NAME}</a>"
 }
 
+def cleanWorkspace (KEEP_WORKSPACE) {
```

Comment:
Minor opinion, I _think_ if we have KEEP_WORKSPACE default to false, then you don't have to pass `false` in the first call below.
`def cleanWorkspace (KEEP_WORKSPACE=false)`

Reply:
If the user requests KEEP_WORKSPACE the initial clean would not be performed which could lead to an issue if there is content on disk.